src/snowcli/cli/nativeapp/commands.py

```diff
@@ -71,24 +64,27 @@ def app_init(
 
 @app.command("bundle", hidden=True)
 @with_output
+@with_project_definition("native_app")
```

Comment:
Implicit in this is that no command will need to grab multiple top-level objects of the project definition. That makes sense to me for now; we can revisit if we find a compelling use case.

Reply:
Correct, in future we can make this argument optional so the command will get whole definition